### DISCUSSION TITLE
Cannot deserialize model error

### DISCUSSION THREAD

**Discussion by sonynavdeep81** [explosion/spaCy#10024] [Help: Coding & Implementations] [answered]
I am trying to load the fasttext vectors **"cc.pa.300.bin"** and for the same I am giving the path for the vectors in **config.cfg** but when I am trying to train the model, I get the following error;

![Screenshot (4)](https://user-images.githubusercontent.com/38432505/148909110-b1753060-997d-4c48-bb97-5b970127e987.jpg)

Please help me, is it because spaCy is expecting vectors in some other format? Kindly help me resolve this problem. Thank You.

**Comment by adrianeboyd** [comment, marked as answer]
The vectors should be loaded into a spacy pipeline using `spacy init vectors` and then the model name (if installed) or path should be used for this setting: https://spacy.io/api/cli#init-vectors

**Reply by sonynavdeep81** [reply]
In this command `python -m spacy init vectors lang vectors_loc output_dir` , it asks for `lang` but my language is **punjabi** which spaCy currently does not support though **"cc.pa.300.bin"** contains **punjabi** vectors. Shall I use `en` then? If I use `en` then it gives the following error;

<img width="587" alt="Screenshot_3" src="https://user-images.githubusercontent.com/38432505/148978075-91583cb2-35c5-4403-ae5b-39e26a8fa21d.png">

The exact command I am using is following and getting the above output;

<img width="364" alt="Screenshot_4" src="https://user-images.githubusercontent.com/38432505/148978506-22dfaeb9-02b5-4374-8f18-d649d1e5df52.png">

Please help!

**Reply by adrianeboyd** [reply]
`init vectors` only loads vectors in word2vec text format (with the size header line), so it can't load from fasttext `.bin` format.

**Reply by sonynavdeep81** [reply]
Yeah got you but is it ok to run this command with `en` although the vectors belong to another language?

**Reply by adrianeboyd** [reply]
You should use the same language here that you intend to use in your config later. For unsupported languages I would typically recommend `xx`, but check that the tokenizer defaults work well enough.

You can use any language where the tokenizer defaults work well (and adjust the tokenizer settings as necessary), but be aware that some of the other languages defaults will be incorrect (stop words, lemmatizer, etc.).
